### ISSUE TITLE
Update programs to QISKit v0.6.x

### ISSUE THREAD

**Issue by psyhtest** [ctuning/ck-qiskit#7]
Two days before the [2nd QCK hackathon](https://www.meetup.com/London-Quantum-Computing-Meetup/events/254156028/) IBM released QISKit v0.6.0 (and v0.6.1 since!)

It didn't take us long to spot one problem with v0.6.0:
```
$ ck run program:qiskit-demo --cmd_key=quantum_coin_flip
...
    * tmp-stderr.tmp

      Traceback (most recent call last):
        File "../quantum_coin_flip.py", line 27, in <module>
          from qiskit import QuantumProgram, QISKitError,
available_backends, register
      ImportError: cannot import name 'QuantumProgram'
```
So we had to fix the QISKit version to 0.5.7 for the hackathon.

In a couple of weeks, we should deal with this API change (`QuantumProgram`) and potentially others (e.g. login changes).

**Comment by psyhtest** [comment]
We should also rename the program to `vqe-qiskit`.

**Comment by psyhtest** [comment]
@ens-lg4 Thanks for updating VQE for the [1st Open QCK Challenge](https://github.com/ctuning/ck-quantum/tree/master/module/challenge.vqe)!

Alas, it's becoming a tradition that just two days before us opening the challenge, IBM released [Qiskit v0.7.0](https://github.com/Qiskit/qiskit-terra/releases/tag/0.7.0) :)
